valkey.conf

```diff
@@ -1908,30 +1908,58 @@ aof-timestamp-enabled no
-
-# The server Slow Log is a system to log queries that exceeded a specified
-# execution time. The execution time does not include the I/O operations
-# like talking with the client, sending the reply and so forth,
-# but just the time needed to actually execute the command (this is the only
-# stage of command execution where the thread is blocked and can not serve
-# other requests in the meantime).
-#
-# You can configure the slow log with two parameters: one tells the server
+################################## COMMAND LOG ###################################
+
+# The Command Log system is used to record commands that consume significant resources
+# during server operation, including CPU, memory, and network bandwidth.
+# These commands and the data they access may lead to abnormal instance operations,
+# the commandlog can help users quickly and intuitively locate issues.
+# Currently, there are three types of commandlogs supported: those used to record
+# commands with long execution times, those with large requests, and those with
+# large query replies.
+
+# The first type is SLOW, log queries that exceeded a specified execution time.
+# The execution time does not include the I/O operations like talking with the client,
+# sending the reply and so forth, but just the time needed to actually execute the
+# command (this is the only stage of command execution where the thread is blocked
+# and can not serve other requests in the meantime).
+#
+# You can configure the slow threshold with two parameters: one tells the server
 # what is the execution time, in microseconds, to exceed in order for the
 # command to get logged, and the other parameter is the length of the
 # slow log. When a new command is logged the oldest one is removed from the
 # queue of logged commands.
-
+#
 # The following time is expressed in microseconds, so 1000000 is equivalent
 # to one second. Note that a negative number disables the slow log, while
 # a value of zero forces the logging of every command.
 slowlog-log-slower-than 10000
-
 # There is no limit to this length. Just be aware that it will consume memory.
-# You can reclaim memory used by the slow log with SLOWLOG RESET.
+# You can reclaim memory used by the slow log with SLOWLOG RESET or COMMANDLOG RESET SLOW.
 slowlog-max-len 128
 
+# The second type is LARGE-REQUEST, which is used to log commands with large requests.
+# Which can consume network bandwidth and the client's query buffer. The request
+# size of a command includes the command itself and all its arguments. For example,
+# in the command SET KEY VALUE, if the key or value is relatively large and causes
+# the overall command size to exceed the set threshold, it will be recorded.
+#
+# The threshold for command request size can be adjusted using the config below, with the
+# unit being bytes.
+large-request-larger-than 1024
+# Record the number of commands.
+large-request-max-len 128
```

Comment:
Are we sure about the default value? Isn't it quite low?

Reply:
you mean large-request-larger-than?

Reply:
Is 1KB a good default for both input/output? Lot's of command response can easily exceed 1KB response I believe.

Reply:
we can discuss it further, my thought is to work backward from the results. In my view, 100MB/s (nearly 1Gbps) is a large amount of traffic. With this level of traffic, I believe the ops that valkey can provide are on the order of 100,000. Therefore, calculating it backward, a default value of 10KB seems more reasonable.